src/main/java/sx/blah/discord/handle/impl/obj/User.java

```diff
@@ -38,6 +38,17 @@
 public class User implements IUser {
 
 	/**
+	 * Default avatars for users
+	 */
+	private static String[] DEFAULT_AVATARS = {
```

Comment:
Pointing to/hardcoding internal assets is iffy. They can often be relocated as has happened in the past.